node-training-assignment/src/user/use-case/create-user-use-case.test.js

```diff
@@ -0,0 +1,46 @@
+let mongoose = require("mongoose");
+let Book = require('../user-model');
+
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+let server = require('../../server');
+let mongoDB = require('../../db');
+
+let should = chai.should();
+
+chai.use(chaiHttp);
+
+describe('Books', () => {
+
+    describe('/POST book', () => {
```

Comment:
This is an api leverl test. Please write a use-case test. For a use-case test, we should just test the use-cases.
eg: if the use -case is to get all users, We just need to test that part. We should not call the api and test the response, request and so on.